{"name": "test_source_profiling_tc01", "status": "broken", "statusDetails": {"message": "TypeError: expected str, bytes or os.PathLike object, not DataFrame", "trace": "csv_source_s1 =     ID   Name               Address        Email  Age Gender\n0  1.0   AAAA  23, asdassdas,adaads  abc@abc.com   34    ... 6.0  AAAFA  28, asdassdas,adaads  abh@abc.com   37      M\n6  7.0   AAAG  29, asdassdas,adaads  abi@abc.com   80      A\n\n    @allure.feature('test_module_01')\n    @allure.story('test_story_01')\n    @allure.severity('normal')\n    @pytest.mark.run(order=1)\n    def test_source_profiling_tc01(csv_source_s1):\n        # logging.info('started data profiling')\n        \"\"\"\n            Use case description: This is a use case description, Test case 01, csv_s1.csv data profiling\n        \"\"\"\n        profiling_tc01 = profilingBase(csv_source_s1)\n>       profiling_tc01.sourcedataprofiling()\n\ntest_tc1_execute_testcsv.py:56: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nbaseconfig_dataProfiling\\dataprofiling_base.py:16: in sourcedataprofiling\n    file_df = os.path.basename(self.df)\n..\\..\\appdata\\local\\programs\\python\\python38\\lib\\ntpath.py:216: in basename\n    return split(p)[1]\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\np =     ID   Name               Address        Email  Age Gender\n0  1.0   AAAA  23, asdassdas,adaads  abc@abc.com   34    ... 6.0  AAAFA  28, asdassdas,adaads  abh@abc.com   37      M\n6  7.0   AAAG  29, asdassdas,adaads  abi@abc.com   80      A\n\n    def split(p):\n        \"\"\"Split a pathname.\n    \n        Return tuple (head, tail) where tail is everything after the final slash.\n        Either part may be empty.\"\"\"\n>       p = os.fspath(p)\nE       TypeError: expected str, bytes or os.PathLike object, not DataFrame\n\n..\\..\\appdata\\local\\programs\\python\\python38\\lib\\ntpath.py:185: TypeError"}, "description": "\n        Use case description: This is a use case description, Test case 01, csv_s1.csv data profiling\n    ", "start": 1606634183985, "stop": 1606634183985, "uuid": "0d4a48c7-1bb2-405e-89eb-cdc71ea96591", "historyId": "fb8a40eb112974914fbadd3d81f4497f", "testCaseId": "e9e96ea69ddfad43bc3a83925482dc57", "fullName": "test_tc1_execute_testcsv#test_source_profiling_tc01", "labels": [{"name": "story", "value": "test_story_01"}, {"name": "severity", "value": "normal"}, {"name": "feature", "value": "test_module_01"}, {"name": "tag", "value": "run(order=1)"}, {"name": "suite", "value": "test_tc1_execute_testcsv"}, {"name": "host", "value": "ZSRIL-1X760Z2"}, {"name": "thread", "value": "4340-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_tc1_execute_testcsv"}]}